src/devices/video/hd44780.h

```diff
@@ -141,6 +143,24 @@ class hd44780_device : public device_t
 	enum        { DDRAM, CGRAM };
 };
 
+// ======================> hd44780u_a00_device
+
+class hd44780u_a00_device :  public hd44780_device
+{
+public:
+	// construction/destruction
+	hd44780u_a00_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+};
+
+// ======================> hd44780u_a02_device
+
+class hd44780u_a02_device :  public hd44780_device
+{
+public:
+	// construction/destruction
+	hd44780u_a02_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+};
```

Comment:
Shouldn’t at least some of these implement `parent_rom_device_type`?

Reply:
These 2 new additions each have a unique char rom.

Reply:
Yeah, but one of the device seems to have an identical ROM, and making the whole group of devices use a single parent would reduce the overall file count in merged sets for people who are into that kind of thing.

Reply:
The KS0066_F00 device already existed and already had its own ROM, which WAS different from the hd44780a00 ROM, but this turned out to be an error (the two are actually supposed to be identical). I changed it so (correctly) both devices now use the same ROM, but (to prevent user annoyance) left the two ROMs as separate files.

Reply:
But it *is* going to cause user annoyance because they’ll need to update their ks0066_f00.zip now, and it won’t be able to share the HD44780A00’s copy of that ROM because there’s no relationship between the devices.